content/other/phaseangles/_meta/collection.json

```diff
@@ -0,0 +1,6 @@
+{
+  "slug": "phaseangles",
+  "title": "Phase Angles",
+  "about": "This collection contains phase angles for a variety of functions, allowing their usage with algorithms like QSVT, QSP, and GQSP.",
```

Comment:
right now it is just one function and there is not GQSP case. Mention it just in case

Reply:
We could release without the collection and add it later when we have more functions.

Reply:
Updated the text :+1: